examples/return_err/lib.rs

```diff
@@ -0,0 +1,75 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
I think this belongs in the `mother` example, I don't think we need an example for each feature, this is covered by the `pass` UI tests.